records_mover/url/optimizer/s3_copy_via_aws_cli.py

```diff
@@ -0,0 +1,22 @@
+
+from contextlib import contextmanager
+from records_mover.url.base import BaseDirectoryUrl
+import logging
+from .gcp_data_transfer_service import GcpDataTransferService
+from typing import Iterator, Tuple, TYPE_CHECKING
+if TYPE_CHECKING:
+    from records_mover.url.s3.s3_directory_url import S3DirectoryUrl
+    from records_mover.url.filesystem import FilesystemDirectoryUrl
+
+logger = logging.getLogger(__name__)
+
+
+class S3CopyViaAwsCli:
+    def copy(self,
+             loc: 'S3DirectoryUrl',
+             other_loc: 'FilesystemDirectoryUrl') -> bool:
+        from .awscli import aws_cli
+
+        # TODO: Can I pass in creds?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker